### PR TITLE
Fix: replace identityServiceURL with host to interact with accounts canister

### PR DESCRIPTION
**Pull request by ghost**
# Motivation

User sees different subaccounts in Svelte and Flutter.

# Changes

* Use the host instead of the identitityServiceUrl for accounts.

# Tests

No changes.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
@bitdivine @peterpeterparker @mstrasinskis please review

File: frontend/svelte/src/lib/constants/host.constants.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const host: string = process.env.HOST as string;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/constants/host.constants.ts:1]: there are additional duplicates of `host: process.env.HOST`, do you plan to replace these too?

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/constants/host.constants.ts:1]: Those are in the `api` folder. I wasn't sure about using something from outside the `api` folder there.

I understand the `api` folder as a standalone module. But I can also change it if that is not the case.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/constants/host.constants.ts:1]: `proposal.api` and `governance.api` are in `api`

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/constants/host.constants.ts:1]: that's what I mean. And this file is not in `api` folder. If we import from this file, then the `api` folder is not independent and could not be used as a standaone module.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/constants/host.constants.ts:1]: but you already do it, look at the change at line 12 of the `account.api.ts` of your PR

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/constants/host.constants.ts:1]: oh, yes, that was already importing from constants, oh, well, then let's do it.

Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/constants/host.constants.ts]: I move it inside `api`, it's only used there anyway.

**Review by peterpeterparker — APPROVED** [review]
LGTM

As discussed `identityServiceURL` was introduced in https://github.com/dfinity/nns-dapp/pull/461/files. It is mandatory for login purpose but for canister better use the proper host.

**Review by bitdivine — APPROVED** [review]

**Pull request merged**